config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presets.yaml

```diff
@@ -23,6 +23,8 @@ presets:
     value: "097f89a0-9286-43d2-9a1a-08f1d49b1af8"
   - name: AZURE_FEDERATED_TOKEN_FILE
     value: "/var/run/secrets/azure-token/serviceaccount/token"
+  - name: PROW_SERVICE_ACCOUNT
+    value: "prowjob-default-sa"
```

Comment:
added `PROW_SERVICE_ACCOUNT` as an env variable to highlight he total set of additions this preset `preset-azure-cred-wi` brings into the test context.